editor/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -192,11 +192,11 @@ class HierarchicalTermSelector extends Component {
 	}
 
 	componentWillUnmount() {
-		if ( this.fetchRequest ) {
+		if ( this.fetchRequest && typeof this.fetchRequest.abort !== 'undefined' ) {
```

Comment:
Minor: maybe lodash result is a good fit here and we can remove all the if condition using it e.g: `result( this.fetchRequest, [ 'abort' ] );`